pallets/qpow/src/tests.rs

```diff
@@ -96,7 +96,7 @@ fn test_submit_valid_proof() {
 }
 
 #[test]
-fn test_verify_for_import() {
+fn test_verify_current_block() {
```

Comment:
it seems this deletes the existing test? We don't need verify for import test anymore?

Reply:
the import and verify paths have been merged